tools/install.py

```diff
@@ -127,7 +129,14 @@ def subdir_files(path, dest, action):
   for subdir, files in ret.items():
     action(files, subdir + '/')
 
-def files(action):
+
+def files(components, action):
+  funcs = {'headers': header_files, 'node': node_files, 'npm': npm_files}
+  for component in components:
+    funcs[component](action)
+
+
```

Comment:
while I like extra newlines, I don't think we're doing that in this file, so this needs single newlines between function declarations